recipes/kyotocabinet/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "kyotocabinet" %}
+{% set version = "1.2.76" %}
+{% set sha256 = "812a2d3f29c351db4c6f1ff29d94d7135f9e601d7cc1872ec1d7eed381d0d23c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://fallabs.com/kyotocabinet/pkg/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [osx]
+
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+    - bzip2
+  run:
+    - zlib
+    - bzip2
+
+test:
+  commands:
+    - test -e $PREFIX/include/kccommon.h  # [unix]
+    - test -e $PREFIX/lib/libkyotocabinet.so  # [linux]
+    - test -e $PREFIX/lib/libkyotocabinet.dylib  # [osx]
+    - test -e $PREFIX/lib/libkyotocabinet.a  # [unix]
+    - test -e $PREFIX/lib/pkgconfig/kyotocabinet.pc  # [unix]
+    - kchashmgr version  # [unix]
```

Comment:
Given as we are only building on Linux, we can strip the selectors and drop the macOS lines.